aspnetcore/migration/Logging-nonaspnetcore.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Migrate from Microsoft.Extensions.Logging 2.1 to 2.2 or 3.0
+author: pakrym
+description: This article outlines the common steps for migrating a non-ASP.NET Core application that uses Microsoft.Extensions.Logging 2.1 to Microsoft.Extensions.Logging 2.2 or 3.0.
+ms.author: pakrym
+ms.custom: mvc
+ms.date: 01/04/2019
+uid: migration/logging-nonaspnetcore
+---
+
```

Comment:
We haven't been putting a blank line here.

Reply:
Blank line ... I actually went thru ages ago and made this consistent. Recommend that we don't put a blank line under the metadata or that we set up a PR to do it for every topic. *My obsessive-compulsive disorder is showing!* :smile: lol